core/services/vrf/vrfcommon/utils.go

```diff
@@ -0,0 +1,66 @@
+package vrfcommon
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+
+	txmgrcommon "github.com/smartcontractkit/chainlink/v2/common/txmgr"
+	txmgrtypes "github.com/smartcontractkit/chainlink/v2/common/txmgr/types"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/txmgr"
+)
+
+type respCountEntry struct {
```

Comment:
Should this be a public type given that it's returned from an exported function?